src/main/java/com/ericsson/ei/mongo/MongoDBHandler.java

```diff
@@ -373,35 +373,12 @@ private boolean doDrop(String dataBaseName, String collectionName, MongoQuery qu
         try {
             db = mongoClient.getDatabase(dataBaseName);
             collectionList = db.listCollectionNames().into(new ArrayList<String>());
-        } catch (MongoCommandException e) {
-            LOGGER.error(
-                    "MongoCommandException, Something went wrong with MongoDb connection. Error: "
-                            + e.getErrorMessage(),
-                    e);
+        } catch (MongoInterruptedException | MongoSocketReadException | MongoSocketWriteException
```

Comment:
I hope the specified exceptions are the all possible exceptions which can be thrown by the try block. Otherwise Exception can be a reasonable choice.

Reply:
Can't answer that, but I guess that these errors should be handle here, whereas other error should be handled somewhere else.